Echo.json5

```diff
@@ -37,6 +37,7 @@
     { name: 'AREnableOnlyTargetSupplyConsignments', value: true },
     { name: 'AREnableAuctionResultsKeywordFilter', value: true },
     { name: 'ARHomeAuctionResultsByFollowedArtists', value: true },
+    { name: 'AREnableNewOnboardingFlow', value: true },
```

Comment:
😬 `true`? did we have this as launch blocking? maybe we should start with `false`?

Reply:
good point, maybe better then to make this a draft until we decide later.

Reply:
we could still add it, but with false for now. later, when we are ok to release, we make it true. echo is like a 2min change from PR to deploy anyway.

Reply:
sorry just saw the comment now, that makes sense but I think it's fine now since we didn't release it anyway :)